app/index.js

```diff
@@ -133,25 +133,27 @@ fs.readdirSync(__dirname + '/routes').forEach(function (file) {
 // Bootsrap addons, like default webGUI
 var AddonManager = require('./addons');
 global.AddonManager = new AddonManager(app, server, io);
```

Comment:
maybe we doesn't need at all - it's not good approach anyway.. if its not used anywhere else just use local variable or perhaps even singleton instance created inside `/addons/index.js`..